Examples/apps/DestinationsExample/DestinationsExample/ViewController.swift

```diff
@@ -44,6 +45,9 @@ class ViewController: UIViewController {
     private var keysFields = [UITextField]()
     private var propertiesFields = [UITextField]()
     
+    var deepLinkData: DeepLink? = nil
+        
+        
     override func viewDidLoad() {
```

Comment:
This appears to be the only change to this file, is it necessary?  Seems like not.

Reply:
ah yeah this is something i implemented to test deep links and missed when i was cleaning up.